### DISCUSSION TITLE
Rodauth OAuth Articles 404

### DISCUSSION THREAD

**Discussion by doloboyz** [HoneyryderChuck/rodauth-oauth#4] [General]
Hello,
I was trying to read the articles linked in README, however, all of the articles seem to have been taken down. By any chance would it be possible to reupload those articles for reference. Since there isn't many resources to read up on or learn about rodauth-oauth other than the docs, it would be nice for new prospective rodauth-oauth users to have access to those articles.

Thanks a lot!

**Comment by HoneyryderChuck** [comment]
Apologies for the late reply, seems that I don't get notifies about discussions yet.

I've updated the links in the readme, thx for the nudge.

**Comment by HoneyryderChuck** [comment]
Apologies for the late reply, seems that I don't get notifies about discussions yet.

I've updated the links in the realmente, thx for the nudge.
